README.md

```diff
@@ -4,221 +4,501 @@
 [![GoDoc](https://godoc.org/github.com/v3io/frames?status.svg)](https://godoc.org/github.com/v3io/frames)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-V3IO Frames is a high-speed server and client library for accessing time-series (TSDB), NoSQL, and streaming data in the [Iguazio Data Science Platform](https://www.iguazio.com).
+V3IO Frames (**"Frames"**) is a multi-model open-source data-access library, developed by Iguazio, which provides a unified high-performance DataFrame API for working with data in the data store of the [Iguazio Data Science Platform](https://www.iguazio.com) (**"the platform"**).
 
-## Documentation
+#### In This Document
 
-Frames currently supports 3 backends and basic CRUD functionality for each.
+- [Client Python API Reference](#client-python-api-reference)
+- [Contributing](#contributing)
+- [LICENSE](#license)
 
-Supported Backends:
-1. TSDB
-2. KV
-3. Stream
-4. CSV - for testing purposes
+<a id="client-python-api-reference"></a>
+## Client Python API Reference
 
+- [Overview](#overview)
+- [User Authentication](#user-authentication)
+- [Client Constructor](#client-constructor)
+- [Common Client Method Parameters](#client-common-method-params)
+- [The create Method](#method-create)
```

Comment:
I Would just call it `Create`, `Write`, `Read`, etc... (instead of "the create method")

Reply:
@talIguaz  I'd rather leave it as-is, because this is reference documentation and the current format clarifies the data type documented in each section + it's consistent with our current doc-site reference doc convetions. (Note that the method names are written with lowercase letters because these are their actual names in the code.)